CHANGELOG.md

```diff
@@ -11,6 +11,10 @@
 - Fixed a bug where imports from permanent stage locations were ignored for temporary stored procedures, UDTFs, UDFs, and UDAFs.
 - Revert back to using CTAS (create table as select) statement for `Dataframe.writer.save_as_table` which does not need insert permission for writing tables.
 
+### New Features
+- Added support for Literal expression with VariantType, for example `Column(Literal(10, VariantType()))` will create a columnar expression corresponding to a SQL literal `10::VARIANT`.
```

Comment:
I still don't think you need to mention it, because `Literal` class is not a public API for users

Reply:
do you have a suggestion on how to mention this change better?

Reply:
hmm I will regard it as an internal change, and do not mention anything in changelog